src/elli/database/refractive_index_info.py

```diff
@@ -23,6 +24,10 @@
 from ..dispersions.base_dispersion import Dispersion, DispersionSum
 from ..materials import IsotropicMaterial
 
+wl_filter_type = Union[
```

Comment:
I think the naming convention for types is pascal, so it should rather be WavelengthFilterType or so.